docs/gapi/gapi_intro.md

```diff
@@ -0,0 +1,52 @@
+# Introduction to OpenCV Graph API (G-API) {#openvino_docs_gapi_gapi_intro}
```

Comment:
why do we need all this stuff in our docs?  It's OpenCV module and all documentation must be there

Reply:
CC @dmatveev @alalek

Reply:
The G-API documentation was integrated by request from Dmitry @dmatveev .

Reply:
ok, question is redirected to @dmatveev - why do we need this docs inside OpenVINO? Such intro must be a part of OpenCV docs, we can just refer to it, I suppose

Reply:
Customers asked for some starting point at the OpenVINO docs as G-API was exposed and served there under the OpenVINO umbrella.

OpenCV docs have this already & the operation/kernel reference will remain there. The OpenVINO-side part will be more Intel/OpenVINO-specific as
- more *PU support comes in the future;
- more OMZ samples get ported on G-API.

Comparison with core/imgproc is not very correct as those parts are quite stable, well known, and not evolve _that_ fast as we do.